data/strings/strings.txt

```diff
@@ -14061,6 +14061,11 @@
     zh-Hans = 该地点的名称
     zh-Hant = 該地點的名稱
 
+  [editor_default_language]
+    tags = android
```

Comment:
```suggestion
    tags = android
    comment = The second part of the editor_edit_place_name_hint to explain that name should be entered in a local language, see https://wiki.openstreetmap.org/wiki/Key:name
```

Reply:
Is there some way to keep the correct capitalisation in the `translate.py` tool?

"de = Der name des ortes, wie er in der landessprache geschrieben wird" is actually
"de = Der Name des Ortes, wie er in der Landessprache geschrieben wird"

Reply:
Yes, that bug was already fixed in `translate.py`:
```
    en = The name of the place, as it is written in the local language
    af = Die naam van die plek, soos dit in die plaaslike taal geskryf is
    ar = اسم المكان كما هو مكتوب باللغة المحلية
    az = Yerin yerli dildə yazıldığı kimi adı
    be = Назва месца, як напісана на мясцовай мове
    bg = Името на мястото, както се изписва на местния език
    ca = El nom del lloc, tal com està escrit en la llengua local
    cs = Název místa, jak je zapsán v místním jazyce
    da = Stedets navn, som det er skrevet på det lokale sprog
    de = Der Name des Ortes, wie er in der Landessprache geschrieben wird
    el = Το όνομα του τόπου, όπως γράφεται στην τοπική γλώσσα
    es = El nombre del lugar, tal y como está escrito en la lengua local
    es-MX = El nombre del lugar, tal como está escrito en el idioma local.
    et = Koha nimi, nagu see on kirjutatud kohalikus keeles.
    eu = Lekuaren izena, bertako hizkuntzan idatzita dagoenez
    fa = نام محل همانطور که به زبان محلی نوشته شده است
    fi = Paikan nimi, kuten se kirjoitetaan paikallisella kielellä.
    fr = Le nom du lieu, tel qu'il est écrit dans la langue locale.
    he = שם המקום, כפי שהוא כתוב בשפת המקום
    hi = स्थान का नाम, जैसा कि स्थानीय भाषा में लिखा गया है
    hu = A helység neve, ahogyan azt a helyi nyelven írják
    id = Nama tempat, seperti yang ditulis dalam bahasa lokal
    it = Il nome del luogo, così come è scritto nella lingua locale
    ja = 現地の言葉で書かれた地名
    ko = 현지 언어로 작성된 장소 이름
    lt = Vietovės pavadinimas, užrašytas vietine kalba
    mr = स्थानिक भाषेत लिहिल्याप्रमाणे ठिकाणाचे नाव
    nb = Navnet på stedet, slik det skrives på det lokale språket.
    nl = De naam van de plaats, zoals die in de plaatselijke taal wordt geschreven
    pl = Nazwa miejsca zapisana w lokalnym języku
    pt = O nome do local, tal como está escrito na língua local
    pt-BR = O nome do lugar, como está escrito no idioma local
    ro = Numele locului, așa cum este scris în limba locală
    ru = Название места, как оно пишется на местном языке
    sk = Názov miesta, ako sa píše v miestnom jazyku
    sv = Platsens namn, som det är skrivet på det lokala språket
    sw = Jina la mahali, kama limeandikwa katika lugha ya ndani
    th = ชื่อสถานที่ตามที่เขียนเป็นภาษาท้องถิ่น
    tr = Yerel dilde yazıldığı şekliyle yerin adı
    uk = Назва місця, як вона пишеться місцевою мовою
    vi = Tên của địa điểm, như được viết bằng ngôn ngữ địa phương
    zh-Hans = 用当地语言书写的地名
    zh-Hant = 用當地語言書寫的地點名稱
```